sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -157,6 +165,27 @@ _rom_ext_start_boot:
   la   t0, (_rom_ext_interrupt_vector + 1)
   csrw mtvec, t0
 
```

Comment:
I think we need to setup virtual addressing before touching the interrupt configuration. The reason being that the interrupt vector needs a C runtime and will be using the virtual addresses. I think we'll need to continue using the mask ROM's interrupt handler initially until we are ready to jump to `rom_ext_main`.